run-local.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+sbt ~run
```

Comment:
This will start the service on port 9000. 
This is an MDTP service, so you should add reserve a port in service-manager-config repo, and then start the service on that port... maybe later, but we should move to this mode eventually